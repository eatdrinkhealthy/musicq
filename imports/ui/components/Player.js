// @flow
import React from "react";
import ReactPlayer from "react-player";

const Player = (): React.Element<*> => (
  <div>
    <ReactPlayer url="https://www.youtube.com/watch?v=uUcEGOLfUTE" playing={false} controls />
    <h3>Requests</h3>
  </div>
);

export default Player;
