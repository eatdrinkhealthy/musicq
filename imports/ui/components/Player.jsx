// @flow
import React from "react";
import ReactPlayer from "react-player";

const Player = (): React$Element<*> => (
  <div>
    <br />
    <div className="row">
      <div className="eight columns offset-by-one">
        <div className="flex-video player">
          <ReactPlayer url="https://www.youtube.com/watch?v=uUcEGOLfUTE" playing={false} controls />
        </div>
      </div>
    </div>
    <div className="row">
      <div className="eight columns offset-by-one">
        <h4>Requests</h4>
      </div>
    </div>
  </div>
);

export default Player;
