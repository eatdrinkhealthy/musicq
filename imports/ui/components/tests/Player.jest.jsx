// @flow
/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */
/* eslint-disable import/no-extraneous-dependencies */
import React from "react";
import renderer from "react-test-renderer";
import Player from "../Player";

describe("<Player />", function () {
  it("matches rendered snapshot", function () {
    const tree = renderer.create(<Player />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
