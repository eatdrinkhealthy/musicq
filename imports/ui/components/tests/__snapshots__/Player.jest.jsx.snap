exports[`<Player /> matches rendered snapshot 1`] = `
<div>
  <br />
  <div
    className="row">
    <div
      className="eight columns offset-by-one">
      <div
        className="flex-video player">
        <div
          className={undefined}
          hidden={false}
          style={
            Object {
              "height": 360,
              "width": 640,
            }
          }>
          <div
            style={
              Object {
                "display": "block",
                "height": "100%",
                "width": "100%",
              }
            }>
            <div />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="row">
    <div
      className="eight columns offset-by-one">
      <h4>
        Requests
      </h4>
    </div>
  </div>
</div>
`;
